csrc/aio/py_lib/deepspeed_py_aio_handle.cpp

```diff
@@ -173,7 +173,7 @@ void deepspeed_aio_handle_t::_stop_threads()
 
 int deepspeed_aio_handle_t::wait()
 {
-    assert(_num_pending_ops > 0);
+    // assert(_num_pending_ops > 0);
```

Comment:
Why disable this assert?

Reply:
I canceled this disabling. It was model related issue.